collections/_posts/2022-06-28-MomentumResidualNeuralNetwork.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: review
+title: "Momentum Residual Neural Networks"
+tags: Invertible ResNets
+author: "Stéphanie Jehan-Besson"
+cite:
+    authors: "Michael E. Sander, Pierre Ablin, Mathieu Blondel, Gabriel Peyré"
+    title:   "Momentum Residual Neural Networks"
+    venue:   "ArXiv, publiseh in the 38th International Conference on Machine Learning, Aug 2021, Online, United States. pp.9276-9287."
+pdf: "https://arxiv.org/pdf/2102.07870.pdf" 
+---
+
+<!-- Do not include the following disclaimer in your own review -->
+
+# Reminders on Residual Neural Networks
+* Residual blocks $x_{n+1}=x_n+f(x_n,\theta_n)$ with a scalar Loss $L(x_n,\theta)$ to minimize
```

Comment:
Hi Stephanie,
To be able to edit latex equation, you should use double symbol $$ before and after the equation, for instance:
$$x_{n+1}=x_n+f(x_n,\theta_n)$$. Take care, you should the change every where in your review !

Reply:
Apparemment, cette notation est possible dans mon éditeur markdown très simple (apostrophe) mais ne passe pas en html effectivement. Je vais résoudre ce problème. Merci beaucoup pour les corrections.